### DISCUSSION TITLE
Import Issue

### DISCUSSION THREAD

**Discussion by AmbroseH** [snipe/snipe-it#13160] [Q&A]
When importing my CSV file, out of the 49 assets I have on that sheet, only 15 are being imported. I've tried all the steps on the common issues page but still have yet to find a solution.

**Comment by marcusmoore** [comment]
Hi there,

We'll need more information to help like what version you are running, any logs that might be helpful, etc.

One step on troubleshooting would be removing the 15 assets that are successfully imported from the sheet, re-importing, and seeing if any of those assets remaining are imported. If they aren't then it is probably a validation issue that the logs might point out.
